homeassistant/components/microbees/binary_sensor.py

```diff
@@ -0,0 +1,92 @@
+"""BinarySensor integration microBees."""
+from microBeesPy import Sensor
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDeviceClass,
+    BinarySensorEntity,
+    BinarySensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import MicroBeesUpdateCoordinator
+from .entity import MicroBeesEntity
+
+BINARYSENSOR_TYPES = {
+    12: BinarySensorEntityDescription(
+        device_class=BinarySensorDeviceClass.MOTION,
+        key="motion_sensor",
+        name="Motion sensor",
```

Comment:
Use [translation_keys](https://developers.home-assistant.io/docs/internationalization/core#entities)

Reply:
how do you suggest use a translation_keys

Reply:
```suggestion
        translation_key="motion_sensor",
```

and this goes to `strings.json`

```
  "entity": {
    "binary_sensor": {
      "motion_sensor": {
        "name": "Motion Sensor"
      }
    }
  }
```

Reply:
Just noticed that you can have placeholders in the translation as well. Would a naming like "{sensor.name} motion sensor" make sense?

Reply:
Just noticed that you can have placeholders in the translation as well. Would a naming like "{sensor.name} motion sensor" make sense?

Reply:
no, actually the user can edit the name from our app, not needed to add the type of device in name

Reply:
I thought that you're supposed add each entity as a device, set name to none, and use has_entity_name = True.

Then the type will automatically be populated.

Reply:
See [entity naming section here](https://developers.home-assistant.io/docs/core/entity)